make all-container in guestbook/php-redis fails

```
[dims@dims-mac 11:27] ~/go/src/k8s.io/examples/guestbook/php-redis ⟩ make all-container
/Applications/Xcode.app/Contents/Developer/usr/bin/make ARCH=amd64 container
cp ./* /var/folders/wr/8hznwwy91lqbbt8b2k99hxtw0000gn/T/tmp.qgOIGGRI
docker build -t gcr.io/google-samples/gb-frontend-amd64:v5 /var/folders/wr/8hznwwy91lqbbt8b2k99hxtw0000gn/T/tmp.qgOIGGRI
Sending build context to Docker daemon  10.75kB
Step 1/10 : FROM php:5-apache
 ---> 12398af6f4de
Step 2/10 : RUN apt-get update
 ---> Using cache
 ---> 9163d32c08a1
Step 3/10 : RUN apt-get install -y php-pear
 ---> Running in 393a8ccd0c49
Reading package lists...
Building dependency tree...
Reading state information...
Package php-pear is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'php-pear' has no installation candidate
The command '/bin/sh -c apt-get install -y php-pear' returned a non-zero code: 100
make[1]: *** [.container-amd64] Error 100
make: *** [sub-container-amd64] Error 2
```

--- a/guestbook/php-redis/Dockerfile
+++ b/guestbook/php-redis/Dockerfile
@@ -15,7 +15,6 @@
 FROM php:5-apache
 
 RUN apt-get update
-RUN apt-get install -y php-pear
 RUN pear channel-discover pear.nrk.io
 RUN pear install nrk/Predis
 

